cmd/delete.go

```diff
@@ -9,9 +9,7 @@ import (
 
 func init() {
 	root.AddCommand(deleteCmd)
-
```

Comment:
It was causing a conflict, right?

Reply:
Conflicts were in the viper registered commands such as "-name" on multiple commands.  This was resolved by only registering the viper connetions on a command's `PreRun`.